xds/third_party/envoy/import.sh

```diff
@@ -80,6 +80,7 @@ envoy/config/core/v3/protocol.proto
 envoy/config/core/v3/proxy_protocol.proto
 envoy/config/core/v3/socket_option.proto
 envoy/config/core/v3/substitution_format_string.proto
+envoy/config/cluster/aggregate/v2alpha/cluster.proto
```

Comment:
Properly sort the list so that we can check existence of a file more easily.

Reply:
It's already sorted 😄

Reply:
My bad, I thought 'o' is before 'l'. 😢 . Fixed now.